storage/replica_command.go

```diff
@@ -623,8 +623,9 @@ func (r *Replica) RangeLookup(batch engine.Engine, args proto.RangeLookupRequest
 	// Checks the results before return as the first range descriptor and the total
 	// range count maybe not correct we have talked above.
 	if args.Reverse {
-		// Checks rds[0]. In case 2 we need to remove first range descriptor.
-		if args.Key.Equal(keys.RangeMetaKey(rds[0].StartKey)) {
+		// We had to carry out a single ascending scan above, and in case two
```

Comment:
I've reworded and removed the reference to the case which belongs to the reverse scan logic further up in the method (outside the diff).